### PR TITLE
Migrate from package:pedantic to package:lints

### LINKED ISSUE #217: at_common_flutter -- Migrate from package:pedantic to package:lints
Describe the solution you'd like:

Per the guidance from dart-lang/lints:

in pubspec.yaml:

-  pedantic: ^1.11.0
+  lints: ^1.0.1

-include: package:pedantic/pubspec.yaml
+include: package:lints/pubspec.yaml

### PR DESCRIPTION
**Pull request by AlgyJr**
**- What I did**
Migrate package: pedantic to package: flutter_lints

**- How I did it**
Changed pedantic dev_dependencies package to lints, changed on `analysis_options.yaml` too (included package).

**- How to verify it**
Checking `pubspec.yaml` and `analysis_options.yaml`
(OR) try to create a variable without camel case type.

**- Description for the changelog**
Migrated package: pedantic to package: flutter_lints

closes #217

### CODE REVIEW

File: at_common_flutter/pubspec.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  pedantic: ^1.11.0
+  lints: ^1.0.1
```
Comment (yahu1031) [inline at_common_flutter/pubspec.yaml]: Hey @Algy-Jr12, Thanks for the PR. How about adding the `lints` package to `dev_dependencies`? Cause Lints are only used while development time, not on runtime. And moreover how about [`flutter_lints`](https://pub.dev/packages/flutter_lints). As this is a flutter widget, `flutter_lints` is a recommended package.

**Comment by AlgyJr** [comment]
Here is it, you can review it and tell me if there are more issues!

**Comment by yahu1031** [comment]
@Algy-Jr12 Yes looks good then how about [`flutter_lints`](https://pub.dev/packages/flutter_lints). As this is a flutter widget, `flutter_lints` is a recommended package.

**Comment by AlgyJr** [comment]
@yahu1031 sorry and thanks once again! How about now?

**Comment by yahu1031** [comment]
@Algy-Jr12 Still buddy, Read the docs clearly to fix this. 😃 take your time no rush buddy.

**Comment by AlgyJr** [comment]
@yahu1031 should I remove the analyzer on `analysis_options.yaml` file? Or is it correct this time?

**Comment by yahu1031** [comment]
> @yahu1031 should I remove the analyzer on `analysis_options.yaml` file? Or is it correct this time?

You read the docs?

**Comment by AlgyJr** [comment]
Yeah

**Comment by yahu1031** [comment]
Then you might have understood that. [Here](https://pub.dev/packages/flutter_lints#usage) check this. Just don't rush buddy. This is fine now. I will approve this. But try reading the docs for better understanding. 😉 Good luck buddy. Keep supporting 💙

**Review by yahu1031 — APPROVED** [review]
Looks good now

**Comment by Algy-Jr12** [comment]
@yahu1031 thanks for the advice and your time helping me! I'll do that!

**Comment by AlgyJr** [comment]
> Then you might have understood that. [Here](https://pub.dev/packages/flutter_lints#usage) check this. Just don't rush buddy. This is fine now. I will approve this. But try reading the docs for better understanding. 😉 Good luck buddy. Keep supporting 💙

@yahu1031 thanks for the advice and your time helping me! I'll do that!

**Comment by yahu1031** [comment]
@Algy-Jr12 [Join our discord server](https://discord.gg/55sHTQFxfz) for more help and info on **The @ platform** packages and libraries 🧡 .

**Review by cpswan — APPROVED** [review]

**Comment by cpswan** [comment]
Thanks @Algy-Jr12 (and thanks @yahu1031 for helping this along)

**Pull request merged**